docs/architecture/notes.md

```diff
@@ -167,29 +178,34 @@ There exist [standard note scripts](https://github.com/0xPolygonMiden/miden-base
 </details>
 
 ## Note storage mode
-Similar to accounts, there are two storage modes for notes in Miden. Notes can be stored on-chain in the [Note DB](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#notes-database) with all data publicly visible for everyone. Alternatively, notes can be stored off-chain by committing only the note hash to the Note DB.
+
+Similar to accounts, there are two storage modes for notes in Miden. Notes can be stored on-chain in the [note database](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#notes-database) with all data publicly visible for everyone. Alternatively, notes can be stored off-chain by committing only the note hash to the note database.
```

Comment:
nit: 

We don't use `on-chain` and `onchain` consistently in this PR. I am not sure if there is a common way in Polygon to do that?

If you also don't know, can you change this `on-chain` to `onchain` (which seems more common in the docs here).